src/Core/SonarLintNotifications/INotificationSettingStorage.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
```

Comment:
Missing copyright headers.
(I would have expected a code smell for this, but for some reason there isn't one)